build/Dockerfile

```diff
@@ -9,9 +9,9 @@ LABEL name="Dynatrace ActiveGate Operator" \
       summary="Dynatrace is an all-in-one, zero-config monitoring platform designed by and for cloud natives. It is powered by artificial intelligence that identifies performance problems and pinpoints their root causes in seconds." \
       description="ActiveGate works as a proxy between Dynatrace OneAgent and Dynatrace Cluster"
 
-ENV OPERATOR=dynatrace-activegate-operator \
```

Comment:
It can stay like this for now, but I do really expect to have more than one Operator Pod to be deployed.